CloudFoundry/content/apps/quotas.md

```diff
@@ -0,0 +1,114 @@
+---
+menu:
+  main:
+    parent: apps
+title: Quotas
+weight: -10
+---
+
+#### Quotas:
+
+Cloud Foundry capacity is managed by quota. Quotas provide a not to exceed reservation of memory, compute, application routes and service instances. 
+
+##### Quota allocations:
+
+We have allocated trial participants a default quota of 10GB of RAM, 1000 routes and 100 service instances.
+
+- 1000 application routes.
+- 10.024GB of RAM.
+- 100 service instances.
+
```

Comment:
Why repeat quotas in text and again in the list again? I'd just leave the list and fix `10.024`GB of ram to `10`GB of RAM